### DISCUSSION TITLE
where is node server entry point running on port 3000 when typed  yarn dev or yarn start?

### DISCUSSION THREAD

**Discussion by dante01yoon** [vercel/next.js#48293] [Help]
### Summary

I'm looking for the server entry point where node server starts on port 3000 in `.next` directory.

I'm using nextjs13.3, but I guess nextjs12, 13 has same server entry point.

when my `next.config.js` set output: "standalone",  I see entrypoint in `.next/standalone/server.js`, but when I erase output: "standalone", I couldn't find entry points.

**Comment by icyJoseph** [comment]
Do you mean this one? https://github.com/vercel/next.js/blob/canary/packages/next/src/server/lib/start-server.ts

When you run `next start` it is a CLI command, that loads up a server, and that one handles requests for you.

**Reply by dante01yoon** [reply]
@icyJoseph Thank you for sharing this!  I'll take a look !
